CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ ___Note:__ Yet to be released changes appear here._
 
 ### General
 
+* `FEAT`: add plugin point for bottom panels ([#3998](https://github.com/camunda/camunda-modeler/pull/3998))
+* `FIX`: integrate DRD find ([#3977](https://github.com/camunda/camunda-modeler/pull/3977))
+* `FIX`: use `RewriteFrames` for path rewrite ([#39999](https://github.com/camunda/camunda-modeler/pull/3999))
```

Comment:
I don't think we need this in the changelog - it is an internal change that helps us inspect Errors in Sentry, not something user facing 
```suggestion
```